live-examples/js-examples/intl/intl-locale.html

```diff
@@ -0,0 +1,12 @@
+<pre>
+<code id="static-js">const korean = new Intl.Locale("ko", {script: "Kore", region: "KR", hourCycle: "h24", calendar: "gregory"});
+const japan = new Intl.Locale("ja-Jpan-JP-u-ca-japanese-hc-h12")
+console.log(korean.baseName)
+// expected output: "ko-Kore-KR"
+
+console.log(japan.baseName)
+// expected output: "ja-Jpan-JP"
```

Comment:
Since my above comments make the example longer, why not compress this bit as you do below, with something like:

```js
console.log(korean.baseName, japan.baseName)
// expected output: "ko-Kore-KR" "ja-Jpan-JP"
```